litestar/handlers/http_handlers/base.py

```diff
@@ -387,6 +389,23 @@ def resolve_after_response(self) -> AsyncCallable | None:
 
         return cast("AsyncCallable | None", self._resolved_after_response)
 
+    def resolve_include_in_schema(self) -> bool | None:
+        """Resolve the 'include_in_schema' property by starting from the route handler and moving up.
+
+        If 'include_in_schema' is found in any of the ownership layers, the last value found is returned.
+        If not found in any layer, the default value ``True`` is returned.
+
+        Returns:
+            bool | None: The resolved 'include_in_schema' property.
+        """
+        if self._resolved_include_in_schema is Empty:
+            include_in_schemas = [
+                i.include_in_schema for i in self.ownership_layers if i.include_in_schema is not Empty
```

Comment:
Wouldn't it be better to check that the value is an instance of bool? We want to filter out none and empty and just have a list of bool values.